assets/_scss/core/_variables.scss

```diff
@@ -18,7 +18,52 @@ $font-serif:        'Georgia', 'Times New Roman', serif;
 $font-normal:       400;
 $font-bold:         700;
 
+
 // Color
+$color-primary:               #0071bc;
+$color-primary-darker:        #205493;
+$color-primary-darkest:       #112e51;
+
+$color-primary-alt:           #02bfe7;
+$color-primary-alt-darkest:   #046b99;
+$color-primary-alt-dark:      #00a6d2;
+$color-primary-alt-light:     lighten($color-primary-alt, 60%); // #9BDAF1
+$color-primary-alt-lightest:  lighten($color-primary-alt, 90%); // #E1F3F8
+
+$color-secondary:             #e31c3d;
+$color-secondary-darkest:     #981b1e;
+$color-secondary-dark:        #cd2026;
+$color-secondary-light:       lighten($color-secondary, 60%); // #e59393
+$color-secondary-lightest:    lighten($color-secondary, 90%); // #f9dede
+
+$color-white:         #ffffff;
+$color-base:          #212121;
+
+$color-gray-dark:         #323a45;
+$color-gray:              lighten($color-gray-dark, 20%); // #205493
+$color-gray-light:        lighten($color-gray-dark, 60%); // #112E51
+$color-gray-lighter:      lighten($color-gray-dark, 80%); // #212121
+$color-gray-lightest:     lighten($color-gray-dark, 90%); // #323A45
+
+$gray-warm-dark:          #494440;
+$gray-warm-light:         lighten($gray-warm-dark, 90%); // #dfdbd9
+$gray-cool-light:         lighten($color-primary, 90%); // #dce4ef
+
+$color-gold:              #fdb81e;
+$color-gold-light:        lighten($color-gold, 20%); // #f9c642
+$color-gold-lighter:      lighten($color-gold, 60%); // #fad980
+$color-gold-lightest:     lighten($color-gold, 83%); // #fff1d2
+
+$color-green:              #2e8540;  
+$color-green-light:        lighten($color-green, 20%); // #649967
+$color-green-lighter:      lighten($color-green, 60%); // #94bfa2
+$color-green-lightest:     lighten($color-green, 80%); // #e7f4e4
+
+$color-cool-blue:          #205493;
+$color-cool-blue-light:    lighten($color-cool-blue, 20%); // #4773aa
+$color-cool-blue-lighter:  lighten($color-cool-blue, 60%); // #8ba6ca
+$color-cool-blue-lightest: lighten($color-cool-blue, 90%); // #dce4ef
+
 $color-grey-50:     #fafafa;
```

Comment:
Are we going to get rid of the rest of these colors once we remove the references in the code?

If so, could you add a "deprecated" comment?